launcher/src/trusted-extensions.ts

```diff
@@ -0,0 +1,60 @@
+/**********************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+import { env } from 'process';
+import { ProductJSON } from './product-json';
+
+export class TrustedExtensions {
+  async configure(): Promise<void> {
+    console.log('# Configuring Trusted Extensions...');
+
+    if (!env.VSCODE_TRUSTED_EXTENSIONS) {
+      console.log('  > env.VSCODE_TRUSTED_EXTENSIONS is not defined, skip this step');
+      return;
+    }
+
+    try {
+      const extensions = env.VSCODE_TRUSTED_EXTENSIONS.split(',');
+      if (!extensions.length) {
+        console.log('  > env.VSCODE_TRUSTED_EXTENSIONS is empty, skip this step');
+        return;
+      }
+
+      for (const e of extensions) {
+        console.log(`  > add ${e}`);
```

Comment:
add to trusted extensions?
I would rephrase it - what and _where_ it's added

Reply:
The whole output in `/checode/entrypoint-logs.txt` should look like this
```
# Configuring Trusted Extensions...
  > add github.copilot
  > add github.copilot-chat
```